db/migrate/20241106150235_change_flipper_gates_value_to_text.rb

```diff
@@ -0,0 +1,16 @@
+# frozen_string_literal: true
+
+class ChangeFlipperGatesValueToText < ActiveRecord::Migration[6.1]
+  def up
+    # Ensure this incremental update migration is idempotent
+    return unless connection.column_exists? :flipper_gates, :value, :string
+
+    remove_index :flipper_gates, %i[feature_key key value] if index_exists? :flipper_gates, %i[feature_key key value]
+    change_column :flipper_gates, :value, :text
+    add_index :flipper_gates, %i[feature_key key value], unique: true, length: { value: 255 }
+  end
+
+  def down
+    change_column :flipper_gates, :value, :string
```

Comment:
Does this also need to do the index modifications to be a true reversal of the above?